slopmop/core/gate_config.py

```
@@ -0,0 +1,101 @@
+   disables the gate. Only an explicit ``false`` disables — an absent or
+   null ``enabled`` leaves the gate on.
+4. Everything else is enabled.
+
+This module is deliberately dependency-free (stdlib only) so any layer can
+import it without joining slop-mop's deferred-import circularity dance.
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any, Dict, Mapping, Tuple, cast
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)